### ISSUE TITLE
fixup PapAnu genetic map

### ISSUE THREAD

**Issue by petrelharp** [popsim-consortium/stdpopsim#1339]
Over at #1338 I've added the PapAnu genetic map to the catalogue... and it turns out that it doesn't actually pass tests, failing with
```
E           AssertionError: SHA256 for _test_cache/tarballs/Pyrho_PAnubis1.0.tar.gz doesn't match the SHA256 for Pyrho_PAnubis1.0. If you didn't add this SHA256 yourself, try deleting _test_cache/tarballs/Pyrho_PAnubis1.0.tar.gz and restarting the tests.
E           assert '63f06ce40715...745f150a6fdd2' == ''
E             + 63f06ce4071508dee8ad130944b95f80d45ead5c53c162f939b745f150a6fdd2
```
Could you look into this, @saurabhbelsare?

**Comment by saurabhbelsare** [comment]
Looks like I had forgotten to add the sha256 to genetic_maps.py before submitting. Sorry about that. I’ve added it now. Should I submit a new PR or how should I add it?

**Comment by petrelharp** [comment]
How about add a new PR incorporating the change in #1338 also? (or, just add to #1338 if you've got permissions)

**Comment by nspope** [comment]
Closed by #1340.
